src/share/index.ts

```diff
@@ -10,6 +10,7 @@ import { shortHash } from "../utils";
 import { ShareType } from "./share-type";
 
 import "../ui/share-popup";
+import { getGCSharePayload } from "../gcinstant";
```

Comment:
this breaks the SDK for third party developers who don't have access to playco packages.

Reply:
I don't believe this package can be built by third parties in any case, because there are multiple private deps. In the bundled output this shouldn't be a problem though

Reply:
Ah I see, we were relying on tree shaking to strip the dep... This already had the unintended side effect of requiring the developer actually is using a toolchain capable of tree shaking though, no?

Reply:
Yeah, that's another issue I've brought up before because it prevents us from ever distributing the SDK on a public CDN.  Right now it depends on the fact that all our templates are using vite and we want everyone on the platform to be using our templates.